main/src/com/google/refine/expr/util/CalenderParser.java

```diff
@@ -491,7 +494,7 @@ void setYear(int val) throws CalendarParserException {
 /**
  * A parser for arbitrary date/time strings.
  */
-public class CalendarParser {
+public class CalenderParser {
```

Comment:
Why was this renamed?
You probably do not want to refer to https://en.wiktionary.org/wiki/calender

Reply:
It is CalendarParser now. a typo but got fixed at later commit.

Reply:
Where is this commit? If it is in this PR, then this typo should not be visible here anymore.

Reply:
I've made a PR on this branch to fix this #1606